frontend/packages/erd-core/src/features/erd/components/ERDRenderer/CommandPalette/CommandPalette.test.tsx

```diff
@@ -155,3 +179,38 @@ describe('preview with option interactions', () => {
     expect(within(preview).getByText('posts')).toBeInTheDocument()
   })
 })
+
+describe('go to ERD with option select', () => {
+  it('go to the table of clicked option and close dialog', async () => {
+    const {
+      user,
+      elements: { dialog },
+      testElements: { activeTableNameDisplay },
+    } = await prepareCommandPalette()
+
+    expect(activeTableNameDisplay).toBeEmptyDOMElement()
+
+    await user.click(within(dialog).getByRole('option', { name: 'follows' }))
+
+    expect(dialog).not.toBeInTheDocument()
+    expect(activeTableNameDisplay).toHaveTextContent(/^follows$/)
```

Comment:
Here, I used RegExp for `toHaveTextContent` matcher to check the whole text.

> When a string argument is passed through, it will perform a partial case-sensitive match to the node content.
> If you want to match the whole content, you can use a RegExp to do it.
https://github.com/testing-library/jest-dom?tab=readme-ov-file#tohavetextcontent